contributing.md

```diff
@@ -48,6 +50,9 @@ gulp is used as a task runner under the hood, but should not be called directly.
 By default, to run the CLI you need to compile the typescript explicitly first. The `ts-node` command line tool will let you run
 `.ts` files directly from the command line. Install it with `npm install -g ts-node`.
 
+The latest `ts-node` required installation in local dev to work
+correctly.
```

Comment:
seems to be a variety of different line lengths in this file - could this be made more consistent at least in the changes?